test/features/timelockDeposit.test.js

```diff
@@ -0,0 +1,29 @@
+const { PoolEnv } = require('./support/PoolEnv')
+
+describe('Re-deposit Timelocked Tokens', () => {
+
+  let env
+
+  beforeEach(() => {
+    env = new PoolEnv()
+  })
+
+  describe('convert timelock to tickets', () => {
+    it('should allow the user to re-deposit timelock as tickets', async () => {
```

Comment:
We could also test converting timelock to sponsorship since "timelockDepositTo" allows specifying the controlled token type

Reply:
Sure, I can add a test